### DISCUSSION TITLE
PHP-FPM seems already running. This might be due to lambda stopped the bootstarp process but didn't leave us oppurtunity to stop PHP-FPM

### DISCUSSION THREAD

**Discussion by tech-emadmin** [brefphp/bref#1223] [Support]
Hi,
We are facing request timeout for all the APIs which are being accessed i.e service unavailable/internal server with 503/500 error.
We are facing this abnormal behavior for the last 40 hours in a test environment. Interestingly, we have deployed the same branch of code in the production environment, which does not encounter this issue.

https://github.com/brefphp/bref/issues/862

We saw some similar discussions with the same issue. Glad to re-open the conversion for a fix.

**Comment by mnapoli** [comment]
Hi, for such a complex issue unfortunately it will all be about collecting as much information as possible to debug.

For example do you see some timeouts? Was anything particular triggering these problems? Any other error in the logs? Any configuration or difference with production? (e.g. VPC, timeout settings, services being used, scaling setup, provisioned concurrency, etc.) How many lambda instances are there with the failures? Does the failure apply to 100% of the lambda instances or just a few of them that are bad? If you scale up or redeploy, do the new lambda instances have the same failure again?
